microsoft-edge/extensions-chromium/whats-new/released-features.md

```diff
@@ -5,17 +5,21 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 05/20/2021
+ms.date: 05/31/2021
 ---
 # Microsoft Edge Add-ons released features
+<!-- suggested new title:
+# Released features for Microsoft Edge Add-ons
+Would need to change at:
+*  # here
+* top of this file (title:) 
+* toc.yml
+* full-text search the repo to update link text that points to the present article
 
-This article describes features launched on [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/) for extension developers.
-
-> [!NOTE]
-> Released features for Microsoft [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/) change often.  Partner Center continues to evolve based on market changes and customer feedback. The features described in this article are also subject to change.
-
-For more information about upcoming updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).
+Consider same for Roadmap article.  But compare title pattern that's used for https://docs.microsoft.com/microsoft-edge/webview2/roadmap
+-->
 
+The following features have been launched on [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/) for developers of Microsoft Edge extensions.
 
 
 <!-- ====================================================================== -->
```

Comment:
Change heading "Developers can respond to user reviews about their Microsoft Edge Add-on" from h3 to h2.  Published:
"https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/released-features#developers-can-respond-to-user-reviews-about-their-microsoft-edge-add-on"

Reply:
Updated the documentation based on the recommended change